app/views/spree/products/show.html.erb

```diff
@@ -1,51 +1,363 @@
+        <%#= render partial: 'properties' %>
       </div>
       <div class="col-12" data-hook="promotions">
-        <%= render partial: 'promotions' %>
+        <%#= render partial: 'promotions' %>
       </div>
     </div>
 
     <div id="no-product-available" class="no-product-available-dropdown">
-      <%= render partial: 'spree/shared/no_product_available' %>
+      <%#= render partial: 'spree/shared/no_product_available' %>
     </div>
 
-    <%= render partial: 'gallery_modal' %>
+    <%#= render partial: 'gallery_modal' %>
   </div>
-  <div id="related-products"></div>
-
-  <%= products_structured_data([@product]) %>
+  <div id="related-products"></div> -->
 
-  <%= render 'spree/shared/product_added_modal' %>
-<% end %>
+  <%#= products_structured_data([@product]) %>
 
+  <%#= render 'spree/shared/product_added_modal' %>
+<%# end %>
+<% @body_id = 'product-details' %>
 <div
   data-related-products
   data-related-products-id="<%= @product.slug %>"
   data-related-products-enabled="<%= @product.respond_to?(:has_related_products?) %>" />
+  <main class="page-contents">
+    <% cache cache_key_for_product do %>
+      <div class="row no-gutters">
+        <div class="col-xl-1 tablet-hide"></div>
+        <div class="col-xl-8 col-lg-8">
+          <div class="product-detail-wrapper">
+              <%= spree_breadcrumbs(@taxon, '', @product) %>
+            <div class="title-wrap position-relative" data-hook="product_show">
+              <h1 class="title"> <%= @product.name %></h1>
+              <div class="ratings">  <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star.svg", alt:'star' %> <%= image_tag "star-off.svg", alt:'star' %><span>5 Reviews</span></div>
+              <ul class="product-standards d-flex flex-wrap align-items-center phone-visible">
+                <li class="glutenfree-icon" data-toggle="tooltip" data-placement="bottom" title="Gluten free">Gluten free</li>
+                <li class="nongmo-icon" data-toggle="tooltip" data-placement="bottom" title="Non GMO">Non GMO</li>
+                <li class="u-icon" data-toggle="tooltip" data-placement="bottom" title="U">u</li>
+                <li class="vegan-icon" data-toggle="tooltip" data-placement="bottom" title="Vegan">Vegan</li>
+              </ul>
+            </div>
+            <div class="product-desc-wrap" > 
+              <div class="row" >
+                <%= render partial: 'gallery' %>
+                <div class="col-xl-5">
+                  <%= render partial: 'phone_cart' %>
+                  <div  id="product-description" >
+                    <%= render partial: 'description' %>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-xl-3 col-lg-4 phone-hide" id="cart-form" data-hook="cart-form" data-hook="product_right_part" data-hook="product_right_part_wrap">
```

Comment:
These hooks (id="cart-form" data-hook="cart-form" ) are not added correctly. It should be like this: id="cart-form" data-hook="cart_form". Please see spree snap

<img width="733" alt="Screen Shot 2021-12-09 at 2 06 24 AM" src="https://user-images.githubusercontent.com/77246192/145284146-b7b18420-040d-4e96-a7bf-d1f5c95a8584.png">